src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonProxy.cs

```diff
@@ -36,7 +36,7 @@ namespace Akka.Cluster.Tools.Singleton
     /// Note that this is a best effort implementation: messages can always be lost due to the distributed nature of the actors involved.
     /// </remarks>
     /// </summary>
-    public sealed class ClusterSingletonProxy : UntypedActor
+    public sealed class ClusterSingletonProxy : ReceiveActor
```

Comment:
Converted to `ReceiveActor` for performance reasons, but mostly to make it easier for me to debug the failing MNTR spec (`PatternMatch` obfuscated the stack trace.)